test/distributed/elastic/agent/server/test/api_test.py

```diff
@@ -298,7 +302,8 @@ def _get_record_metrics_test_calls(
         return calls
 
     def test_rendezvous(self):
-        spec = self._get_worker_spec(max_restarts=1)
+        hostname = _get_fq_hostname()
+        spec = self._get_worker_spec(max_restarts=1, local_addr=hostname)
```

Comment:
I see this test is changing? Are we changing behavior here around requiring the fqdn being passed in?

Reply:
Collectives operation is happening in the handler now so we need a address that is resolvable.
Prior to this changes tests would just query store for master_addr/master_port on the store.